Lib/fontbakery/specifications/googlefonts.py

```diff
@@ -109,7 +109,6 @@
       , 'com.google.fonts/check/075' # Check for points out of bounds.
       , 'com.google.fonts/check/076' # Check glyphs have unique unicode codepoints.
       , 'com.google.fonts/check/077' # Check all glyphs have codepoints assigned.
-      , 'com.google.fonts/check/078' # Check that glyph names do not exceed max length.
```

Comment:
Since we're disabling, please simply comment the line out instead of deleting it.